protostar/cairo/cairo_function_runner_facade.py

```diff
@@ -9,6 +9,9 @@
 
 
 RUNNER_BUILTINS = ["pedersen", "range_check", "bitwise", "ec_op"]
+RUNNER_BUILTINS_CAMEL_CASE = [
```

Comment:
This is not `camelCase` really it's `TitleCase`/`PascalCase`, right?

Reply:
[camel -> title](https://github.com/software-mansion/protostar/pull/1655/commits/870b0e4300799277b974bd83f6806c498189ed74)